packages/app/components/trending/tabs/nfts-list.tsx

```diff
@@ -59,36 +41,23 @@ export const NFTSList = forwardRef<TrendingTabListRef, TrendingTabListProps>(
     );
 
     const renderItem = useCallback(
-      ({ item: chuckItem, index: itemIndex }: ListRenderItemInfo<NFT[]>) => {
+      ({ item, index }: ListRenderItemInfo<NFT>) => {
         return (
-          <View tw="flex-row">
-            {chuckItem.map((item, chuckItemIndex) => (
-              <Card
-                key={item.nft_id}
-                nft={item}
-                onPress={() =>
-                  onItemPress(itemIndex * NUM_COLUMNS + chuckItemIndex)
-                }
-                numColumns={NUM_COLUMNS}
-                style={{
-                  marginRight: StyleSheet.hairlineWidth,
-                  marginBottom: StyleSheet.hairlineWidth,
-                }}
-              />
-            ))}
-          </View>
+          <Card
+            nft={item}
+            onPress={() => onItemPress(index)}
+            numColumns={NUM_COLUMNS}
+          />
```

Comment:
It's preferred to `useCallback` here.
Define onPress as a const with useCallback on line 45.